modules/extension/app-schema/app-schema/src/main/java/org/geotools/appschema/jdbc/JoiningJDBCFeatureSource.java

```diff
@@ -372,7 +384,8 @@ protected static String createAlias(String typeName, Set<String> tableNames) {
     public String selectSQL(
             SimpleFeatureType featureType,
             JoiningQuery query,
-            AtomicReference<PreparedFilterToSQL> toSQLref)
+            AtomicReference<PreparedFilterToSQL> toSQLref,
+            boolean isCount)
```

Comment:
Add to comment what does the isCount parameter.

Reply:
fixed